system/Config/View.php

```diff
@@ -33,13 +33,17 @@ class View extends BaseConfig
      *
      * To prevent potential abuse, all filters MUST be defined here
      * in order for them to be available for use within the Parser.
+     *
+     * @var array<string>
```

Comment:
```suggestion
     * @var array<string, string>
     * @phpstan-var array<string, callable-string>
```

Reply:
Please do the same with `$plugins`, `$corePlugins`, and `$coreFilters`.

Reply:
```log
------ ---------------------------------------------------------------------------------------------- 
  Line   View.php
 ------ ----------------------------------------------------------------------------------------------
  40     Property CodeIgniter\Config\View::$filters type has no signature specified for callable.
  50     Property CodeIgniter\Config\View::$plugins type has no signature specified for callable.
  58     Property CodeIgniter\Config\View::$coreFilters type has no signature specified for callable.
  88     Property CodeIgniter\Config\View::$corePlugins type has no signature specified for callable.
 ------ ----------------------------------------------------------------------------------------------
```

But I get these errors after following the revise.

Reply:
You added `callable-string` and phpstan complains?

Reply:
![image](https://github.com/codeigniter4/CodeIgniter4/assets/65348108/ab78cf5d-48ee-4989-ac6c-279ebe8695bf)

Yes, I actually added it.
Show as above.

Reply:
Okay. Please add them as is. Then add the new errors to baseline by: `vendor/bin/phpstan analyse --generate-baseline phpstan-baseline.php`. I'll do the corrections to phpstan.